lib/graphql_metrics/extractor.rb

```diff
@@ -131,19 +72,21 @@ def extract_batch_loaders
     end
 
     def extract_query
-      return unless respond_to?(:query_extracted)
+      return unless query_extracted_method = extraction_method(:query_extracted)
 
-      start_time = ctx_namespace[START_TIME_KEY]
-      return unless start_time
+      start_time = instrumentor.ctx_namespace[Instrumentation::START_TIME_KEY] if instrumentor
```

Comment:
```suggestion
      if instrumentor
        start_time = instrumentor.ctx_namespace[Instrumentation::START_TIME_KEY]
        return unless start_time
      end
```

Is this the same?

Reply:
That works in terms of readability, but then `start_time` is initialized within the `if/end` block scope, not the parent method.

This is both correct and easier to read: 
```ruby
start_time = if instrumentor
  instrumentor.ctx_namespace[Instrumentation::START_TIME_KEY].tap do |start_time|
    return unless start_time
  end
end
```